### ISSUE TITLE
Unknown rx form 'anychar' when trying to find unlinked references

### ISSUE THREAD

**Issue by AlexanderDavid** [org-roam/org-roam#836]
### Description
When running the command `org-roam-unlinked-references` I get the error `Unknown rx form 'anychar'`
#### Steps to Reproduce
1. Load emacs
2. Go to an org roam file
3. Run `org-roam-unlinked-references`

#### Backtrace

```
Debugger entered--Lisp error: (error "Unknown rx form ‘anychar’")
  signal(error ("Unknown rx form ‘anychar’"))
  error("Unknown rx form `%s'" anychar)
  rx-form(anychar *)
  rx-kleene((one-or-more anychar))
  rx-form((one-or-more anychar))
  rx-submatch((group (one-or-more anychar)))
  rx-form((group (one-or-more anychar)) :)
  #f(compiled-function (x) #<bytecode 0x4a5f35>)((group (one-or-more anychar)))
  mapconcat(#f(compiled-function (x) #<bytecode 0x4a5f35>) ((group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything))) nil)
  rx-and((and (group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything))))
  rx-form((and (group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything))))
  rx-to-string((and (group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything))) t)
  #f(compiled-function (&rest regexps) #<bytecode 0x4a7361>)((group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything)))
  (rx (group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything)))
  (let* ((titles (org-roam--extract-titles)) (rg-command (concat "rg -o --vimgrep -P -i " (string-join (mapcar (lambda (glob) (concat "-g " glob)) (org-roam--list-files-search-globs org-roam-file-extensions)) " ") (format " '\\[([^[]]++|(?R))*\\]%s' " (mapconcat (lambda (title) (format "|(\\b%s\\b)" ...)) titles "")) org-roam-directory)) (file-loc (buffer-file-name)) (buf (get-buffer-create "*org-roam unlinked references*")) (results (split-string (shell-command-to-string rg-command) "\n")) (result-regex (rx (group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything))))) (pop-to-buffer buf) (let ((inhibit-read-only t)) (erase-buffer) (org-mode) (insert (propertize (car titles) (quote font-lock-face) (quote org-document-title)) "\n\n" "* Unlinked References\n") (dolist (line results) (save-match-data (when (string-match result-regex line) (let ((file ...) (row ...) (col ...) (match ...)) (when (and match ... ...) (let ... ... ... ...)))))) (read-only-mode 1) (dolist (title titles) (highlight-phrase (downcase title) (quote bold-italic))) (goto-char (point-min))))
  (if (not (executable-find "rg")) (user-error "Cannot find the \"rg\" executable, aborting") (let* ((titles (org-roam--extract-titles)) (rg-command (concat "rg -o --vimgrep -P -i " (string-join (mapcar (lambda ... ...) (org-roam--list-files-search-globs org-roam-file-extensions)) " ") (format " '\\[([^[]]++|(?R))*\\]%s' " (mapconcat (lambda ... ...) titles "")) org-roam-directory)) (file-loc (buffer-file-name)) (buf (get-buffer-create "*org-roam unlinked references*")) (results (split-string (shell-command-to-string rg-command) "\n")) (result-regex (rx (group (one-or-more anychar)) ":" (group (one-or-more digit)) ":" (group (one-or-more digit)) ":" (group (zero-or-more anything))))) (pop-to-buffer buf) (let ((inhibit-read-only t)) (erase-buffer) (org-mode) (insert (propertize (car titles) (quote font-lock-face) (quote org-document-title)) "\n\n" "* Unlinked References\n") (dolist (line results) (save-match-data (when (string-match result-regex line) (let (... ... ... ...) (when ... ...))))) (read-only-mode 1) (dolist (title titles) (highlight-phrase (downcase title) (quote bold-italic))) (goto-char (point-min)))))
  org-roam-unlinked-references()
  funcall-interactively(org-roam-unlinked-references)
  call-interactively(org-roam-unlinked-references record nil)
  command-execute(org-roam-unlinked-references record)
  counsel-M-x-action("org-roam-unlinked-references")
  ivy-call()
  ivy-read("M-x " ("toggle-debug-on-error" "org-roam-unlinked-references" "org-insert-link" "org-roam-store-link" "deft" "company-org-roam" "doom/reload" "orb-note-actions" "helm-bibtex" "package-install" "eval-region" "load-theme" "toggle-word-wrap" "org-ref-cite-click-helm" "python-docstring/generate-docstring" "org-ref" "org-roam-db-build-cache" "org-ref-helm-insert-cite-link" "org-roam" "package-delete" "fundamental-mode" "org-roam-find-file" "doom/delete-this-file" "org-noter-insert-note" "helm-bibtex-with-notes" "which-key-show-minor-mode-keymap" "find-grep" "hugo-status" "org-roam-insert" "org-roam-find-index" "orb-find-non-ref-file" "org-noter" "python-mode" "org-reftex-citation" "org-roam-graph" "org-footnote-new" "doom-modeline-mode" "org-roam-graph-show" "global-writeroom-mode" "org-roam-dailies-today" "python-docstring-minor-mode" "customize" "evil-mode" "describe-key" "eww-open-file" "org-ref-index" "dired-do-search" "org-roam-capture" "org-roam-tomorrow" "evil-previous-visual-line" ...) :predicate #f(compiled-function (x) #<bytecode 0x150df55>) :require-match t :history counsel-M-x-history :action counsel-M-x-action :keymap (keymap (67108908 . counsel--info-lookup-symbol) (67108910 . counsel-find-symbol)) :initial-input nil :caller counsel-M-x)
  counsel-M-x()
  funcall-interactively(counsel-M-x)
  call-interactively(counsel-M-x nil nil)
  command-execute(counsel-M-x)
```

#### Expected Results
Unlinked references are listed

#### Actual Results
The error listed above pops up

### Environment
- Emacs: GNU Emacs 26.3 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.20)
 of 2020-05-19
- Framework: Doom
- Org: Org mode version 9.4 (nil @ /home/alex/.emacs.d/.local/straight/build/org-mode/)
- Org-roam: 1.2.0

- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/eca07277cee2204a9327f9ab8bf766dd0614cb56

**Comment by zaeph** [comment]
Normally, this was fixed with #812.  Could you update Org-roam and try again?

**Comment by AlexanderDavid** [comment]
That fixed it. I had to run `doom update`! Thanks
